nano/node/bootstrap/bootstrap.cpp

```diff
@@ -875,6 +910,41 @@ void nano::bootstrap_attempt::lazy_backlog_cleanup ()
 	}
 }
 
+void nano::bootstrap_attempt::lazy_destinations_increment (nano::account const & destination_a)
+{
+	// Update accounts counter for send blocks
+	auto existing (lazy_destinations.get<account_tag> ().find (destination_a));
+	if (existing != lazy_destinations.get<account_tag> ().end ())
+	{
+		lazy_destinations.get<account_tag> ().modify (existing, [](nano::lazy_destinations_item & item_a) {
+			++item_a.count;
+		});
+	}
+	else
+	{
+		lazy_destinations.insert (nano::lazy_destinations_item{ destination_a, 1 });
+	}
+}
+
+void nano::bootstrap_attempt::lazy_destinations_flush ()
+{
+	size_t count (0);
+	nano::unique_lock<std::mutex> lazy_lock (lazy_mutex);
```

Comment:
This can be a `std::lock_guard`

Reply:
Replaces here & in a few other functions